app/javascript/views/userDetail/UserDetail.test.js

```diff
@@ -208,10 +208,10 @@ describe('UserDetail', () => {
 
     describe('renders cards', () => {
       it('should display <UserDetailShow/>', () => {
-        wrapper.setProps({ isUserEditable: false })
-        expect(wrapper.find('UserDetailShow').length).toBe(1)
         wrapper.setProps({ isUserEditable: true })
         expect(wrapper.find('UserDetailShow').length).toBe(0)
+        wrapper.setProps({ isUserEditable: false })
```

Comment:
Not sure why the order of these was changed, but they should independent.  Would be better to stitch to two separate 'it' blocks....
```suggestion
        it('should display <UserDetailShow/> when user is not editable', () => {
	        wrapper.setProps({ isUserEditable: false })
	        expect(wrapper.find('UserDetailShow').length).toBe(1)
	      });
	              it('should not display <UserDetailShow/> when user is editable', () => {
	        wrapper.setProps({ isUserEditable: true })
	        expect(wrapper.find('UserDetailShow').length).toBe(0)
	      });
```

Reply:
Updated!